### PR TITLE
Fill missing sections in README

### PR DESCRIPTION
**Pull request by sbarzowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,9 +81,26 @@ See the [contributing page](http://jsonnet.org/contributing.html) on our website
...
-TODO
+```
+make && make libjsonnet.so &&  ./tests.sh
+```
 
 
 ### Locally serving the website
 
-TODO
+First, you need to build JS version of Jsonnet (requires emscripten):
```
Comment (seh) [inline README.md]:
```suggestion
First, you need to build the JavaScript version of Jsonnet (which requires [_emscripten_](https://emscripten.org/)):
```

File: README.md [hunk 2]
```diff
@@ -81,9 +81,26 @@ See the [contributing page](http://jsonnet.org/contributing.html) on our website
...
+```
+make doc/js/libjsonnet.js
+```
+
+Then inside `doc` directory you can generate and serve the website using [Jekyll](https://jekyllrb.com/):
```
Comment (seh) [inline README.md]:
```suggestion
Then, inside the _doc_ directory, you can generate and serve the website using [Jekyll](https://jekyllrb.com/):
```

File: README.md [hunk 3]
```diff
@@ -81,9 +81,26 @@ See the [contributing page](http://jsonnet.org/contributing.html) on our website
...
+```
+jekyll serve --watch --livereload
+```
+Or you can do the same more concisely:
+```
+jekyll s -w -l
+```
+
+This should serve the website, automatically rebuild when you change underlying files and automatically
```
Comment (seh) [inline README.md]:
```suggestion
This should serve the website, automatically rebuild when you change any underlying files, and automatically
```

File: README.md [hunk 4]
```diff
@@ -81,9 +81,26 @@ See the [contributing page](http://jsonnet.org/contributing.html) on our website
 
 ### Running tests
 
-TODO
+```
+make && make libjsonnet.so &&  ./tests.sh
```
Comment (sparkprime) [inline README.md]: You can just do `make test` to get this (and it also builds the little C programs that are tested).

File: README.md [hunk 5]
```diff
@@ -81,9 +81,26 @@ See the [contributing page](http://jsonnet.org/contributing.html) on our website
...
 
 ### Locally serving the website
 
-TODO
+First, you need to build the JavaScript version of Jsonnet (which requires [_emscripten_](https://emscripten.org/)):
+```
+make doc/js/libjsonnet.js
+```
+
+Then, inside the _doc_ directory, you can generate and serve the website using [Jekyll](https://jekyllrb.com/):
```
Comment (sparkprime) [inline README.md]: There is a script `tools/scripts/serve_docs.sh` which does this but without --livereload.  Perhaps it should use --livereload though.  It must be run from the root of the repo (but it has friendly checks for that).

**Comment by sparkprime** [comment]
Thanks, I forked & merged this.

**Comment by sparkprime** [comment]
Oh crap, the version of Jekyll on my work machines does not have --livereload.  I'll have to disable that for now.

**Comment by sbarzowski** [comment]
Oh, thanks, I completely forgot about this one

**Pull request closed without merging**